src/test/java/io/github/externschool/planner/service/VerificationKeyServiceTest.java

```diff
@@ -0,0 +1,99 @@
+package io.github.externschool.planner.service;
+
+import io.github.externschool.planner.entity.VerificationKey;
+import io.github.externschool.planner.repository.VerificationKeyRepository;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class VerificationKeyServiceTest {
+
+    @Mock
+    private VerificationKeyRepository repository;
+
+    @InjectMocks
```

Comment:
https://tedvinke.wordpress.com/2014/02/13/mockito-why-you-should-not-use-injectmocks-annotation-to-autowire-fields/